src/platforms/windows/windowsappimageprovider.h

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef WINDOWSAPPIMAGEPROVIDER_H
+#define WINDOWSAPPIMAGEPROVIDER_H
+
+#include <QObject>
+#include <QQuickImageProvider>
+
+
+class WindowsAppImageProvider final : public QQuickImageProvider,
+                                      public QObject {
+ public:
+  WindowsAppImageProvider(QObject* parent);
+  ~WindowsAppImageProvider();
+  QPixmap requestPixmap(const QString& id, QSize* size,
```

Comment:
`size` is not needed because we can extract it from `QPixmap`. Am I right?

Reply:
It is an out parameter. We need to write the size of the pixmap into it.